src/components/Assets/LockableAssetRow.tsx

```diff
@@ -59,18 +65,9 @@ export const LockableAssetRow = memo<LockableAssetRowProps>(
       () => ({
         title: token.type === "lst" ? "Lock" : "Lock & Stake",
         onClick: () => handleLockClick(token.accountId),
-        disabled:
-          !!stakingPoolId &&
-          token.type === "lst" &&
-          stakingPoolId !== token.accountId,
-        tooltip:
-          !!stakingPoolId &&
-          token.type === "lst" &&
-          stakingPoolId !== token.accountId
-            ? "You can't lock more than one LST."
-            : undefined,
```

Comment:
I believe we need to keep this logic - the contracts enforce that you can only stake with 1 pool at a time and although it's possible to switch staking pools we do not currently support that flow at the moment - it would be out of scope for this PR IMO.